[2024-12-04T16:31:02.104+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-04T16:31:02.128+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: instacart_etl_pipeline.etl_dim_product manual__2024-12-04T16:30:56.681941+00:00 [queued]>
[2024-12-04T16:31:02.141+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: instacart_etl_pipeline.etl_dim_product manual__2024-12-04T16:30:56.681941+00:00 [queued]>
[2024-12-04T16:31:02.143+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-04T16:31:02.165+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_dim_product> on 2024-12-04 16:30:56.681941+00:00
[2024-12-04T16:31:02.179+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=13093) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-04T16:31:02.182+0000] {standard_task_runner.py:72} INFO - Started process 13095 to run task
[2024-12-04T16:31:02.179+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'instacart_etl_pipeline', 'etl_dim_product', 'manual__2024-12-04T16:30:56.681941+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/instacart_etl_dag.py', '--cfg-path', '/tmp/tmp_jmezw2q']
[2024-12-04T16:31:02.183+0000] {standard_task_runner.py:105} INFO - Job 234: Subtask etl_dim_product
[2024-12-04T16:31:02.243+0000] {task_command.py:467} INFO - Running <TaskInstance: instacart_etl_pipeline.etl_dim_product manual__2024-12-04T16:30:56.681941+00:00 [running]> on host e64be71b3b6d
[2024-12-04T16:31:02.364+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='instacart_etl_pipeline' AIRFLOW_CTX_TASK_ID='etl_dim_product' AIRFLOW_CTX_EXECUTION_DATE='2024-12-04T16:30:56.681941+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-04T16:30:56.681941+00:00'
[2024-12-04T16:31:02.365+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-04T16:31:02.391+0000] {base.py:84} INFO - Retrieving connection 'mssql_default'
[2024-12-04T16:33:06.201+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-04T16:33:06.210+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-04T16:33:06.211+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=instacart_etl_pipeline, task_id=etl_dim_product, run_id=manual__2024-12-04T16:30:56.681941+00:00, execution_date=20241204T163056, start_date=20241204T163102, end_date=20241204T163306
[2024-12-04T16:33:06.253+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-04T16:33:06.269+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-04T16:33:06.275+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
